### ISSUE TITLE
Can no longer run on Arch

### ISSUE THREAD

**Issue by LiamDawe** [snapcrafters/obs-studio#122]
Hi, just updated and now it won't run:

> 
> info: OBS 27.0.0-modified (linux)
> info: ---------------------------------
> info: ---------------------------------
> info: audio settings reset:
> 	samples per sec: 48000
> 	speakers:        2
> error: os_dlopen(libobs-opengl.so.0->libobs-opengl.so.0): /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /var/lib/snapd/lib/gl/libEGL.so.1)
> 
> error: Failed to initialize video:  Graphics module not found
> info: Freeing OBS context data
> info: == Profiler Results =============================
>

**Comment by blastermaster77** [comment]
same problem here

**Comment by compsup** [comment]
Same exact problem

**Comment by BackThrowTheBaguetteBrawler** [comment]
this is also happening for me on nvidia (465) on fedora 34

**Comment by diddledan** [comment]
this looks like it might be a confinement compatibility issue with your host opengles library. Could I ask someone to report this on the [forum](https://forum.snapcraft.io) in the `snapd` category indicating that the host opengles library is unable to be loaded due to the OBS snap and its base snap including an older libc library?

**Comment by cglmrfreeman** [comment]
Same problem, any solutions yet? The snap forums have no replies...

**Comment by Retardium** [comment]
Yeah, I've opened [a thread at the snap forums](https://forum.snapcraft.io/t/error-os-dlopen-libobs-opengl-so-0-libobs-opengl-so-0-lib-x86-64-linux-gnu-libc-so-6-version-glibc-2-33-not-found-required-by-var-lib-snapd-lib-gl-libegl-so-1-obs-studio-arch-manjaro/25675)  like recommended, but like you wrote - no replies. 
We only can hope the devs have read it and fix it.

**Comment by cglmrfreeman** [comment]
Actually I've dropped Snap's OBS for a build provided by my distro's Packman repo.

**Comment by arighi** [comment]
Same problem on Ubuntu impish 21.10 (can't use snap obs anymore). I switched to the obs provided by the obsproject ppa.

**Comment by flexiondotorg** [comment]
Fixed snap released as revision 1273 (or newer), in the stable channel.
